Gruntfile.js

```diff
@@ -152,9 +133,6 @@ module.exports = function (grunt) {
     });
   });
 
-  grunt.loadTasks('test/tasks');
-
-  grunt.registerTask('test', ['test:node']);
   grunt.registerTask(
     'test:node',
```

Comment:
this one is also replace by npm script `test:node`, which does the same.
plus, there is no more `grunt mocha` task (see `['build:node', 'mocha']` below), so this command will fail